src/lib/containers/entity/annotations/CustomArrayFieldTemplate.tsx

```diff
@@ -57,11 +54,10 @@ export function CustomArrayFieldTemplate<T>(props: ArrayFieldTemplateProps<T>) {
             <div style={{ display: 'flex' }}>
               <Button
                 variant="primary-500"
-                className="AddButton"
                 onClick={props.onAddClick}
                 type="button"
               >
-                <Add />
+                Add Item <Add />
```

Comment:
Still workshopping w/ Ljubo what the add button should look like when there are no items